hudi-client/hudi-spark-client/pom.xml

```diff
@@ -132,6 +132,14 @@
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hudi</groupId>
+      <artifactId>hudi-timeline-service</artifactId>
```

Comment:
can you help me understand, why are we adding the dep now in this patch? 
I would have expected it to be part of dependency chain long back. If not, why do we need to add it in this patch? can you help throw some light. 

@rmahindra123

Reply:
This is using test classes as test dependency so it is OK.